crates/model/src/quote.rs

```diff
@@ -209,6 +209,8 @@ pub struct OrderQuote {
     pub partially_fillable: bool,
     pub sell_token_balance: SellTokenSource,
     pub buy_token_balance: BuyTokenDestination,
+    #[serde(flatten)]
+    pub signing_scheme: QuoteSigningScheme,
```

Comment:
We should return the `SigningScheme` here and not the `QuoteSigningScheme` (since the additional signing information like `is_onchain_order` and `verification_gas_limit` are only needed for quoting and not order creation).

```suggestion
    pub signing_scheme: SigningScheme,
```